_haikus/jellyfish_cloud.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Jellyfish Cloud
+author: Valerdi
+---
+
+Multiple cloud <br>
+Like the different types of jellyfish<br>
+distributed in different places <br>
```

Comment:
The line breaks are looking great, too! Haikus follow a syllabic format, though, so each line should be a certain number of syllables. 

Your first line should be 5 syllables, the second line 7 syllables, and the last line 5 syllables.

You can check to make sure you have the right number of syllables by using a tool such as [http://www.haikusyllablecounter.com/](http://www.haikusyllablecounter.com/).

Once you've updated the haiku to follow this format, we can merge it!